### PR TITLE
[ES] Update common temperature definition

### PR DESCRIPTION
**Pull request by Kalma-House**
More ways to request a temperature.

### CODE REVIEW

File: tests/es/climate_HassClimateSetTemperature.yaml [hunk 1]
```diff
@@ -2,22 +2,30 @@ language: "es"
 tests:
   - sentences:
       - "pon la temperatura a 30"
+      - "sube la temperatura a 30"
+      - "baja la temperatura a 30"
```
Comment (cibernox) [inline tests/es/climate_HassClimateSetTemperature.yaml:6]: have you considered "configura la temperatura a 30"? I can see myself saying something like that when speaking to my oven or my thermostat. But admittedly I'm a bit of a snob.

File: tests/es/climate_HassClimateSetTemperature.yaml [hunk 2]
```diff
@@ -2,22 +2,30 @@ language: "es"
...
     intent:
       name: "HassClimateSetTemperature"
       slots:
         temperature: 30
   - sentences:
       - "pon la temperatura a 30 grados celsius"
+      - "sube la temperatura a 30 grados celsius"
```
Comment (cibernox) [inline tests/es/climate_HassClimateSetTemperature.yaml:13]: What happens if you omit celsius and fahrenheit? What will it default to?

File: tests/es/climate_HassClimateSetTemperature.yaml (hunk 1 above)
Comment (cibernox) [reply tests/es/climate_HassClimateSetTemperature.yaml:6]: Sure, you can add one extra commit to your branch and the PR will reflect that. Although if you made this using github's web user interface, I'm not sure how to do it. 

In any case, it's not important. I'd cool with merging this as is and keep making improvements later.

Comment (Kalma-House) [reply tests/es/climate_HassClimateSetTemperature.yaml:6]: Found the way. Changes committed. Thanks for your input.

File: tests/es/climate_HassClimateSetTemperature.yaml (hunk 2 above)
Comment (cibernox) [reply tests/es/climate_HassClimateSetTemperature.yaml:15]: Makes sense. At least is what I would expect.

**Comment by Kalma-House** [comment]
I am merging PR [#473](https://github.com/home-assistant/intents/pull/473) and [#474](https://github.com/home-assistant/intents/pull/474) into this one, in the hope that it then passes the tests.

**Comment by davefx** [comment]
Please solve conflicts in order to review and possibly merge this PR

**Review by davefx — CHANGES_REQUESTED** [review]
A typo must be fixed

**Review by davefx — APPROVED** [review]

**Pull request merged**